src/pages/chat/Chat.tsx

```diff
@@ -92,6 +94,14 @@ const Chat = () => {
     }
   }, [selectedChat, messages.length])
 
+  useEffect(() => {
+    if (currentChatId) {
+      listOfChats.forEach((chat: ChatResponse) => {
+        if (chat._id === currentChatId) setSelectedChat(chat)
```

Comment:
As I see it will navigate only if chat exists?

Reply:
> As I see it will navigate only if chat exists?

Yup, that's right

Reply:
It also should create a new chat and navigate to it if the chat doesn`t exist

Reply:
> It also should create a new chat and navigate to it if the chat doesn`t exist

I guess it wouldn't make sense, because we don't even have a button, unless we're talking to the user. 
![зображення](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/503e202e-f185-4fc3-abd8-b0ccf0f8c23c)


And the chat will be created as soon as we send the first message
![зображення](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/93e1602c-b779-455c-89ff-6c26e24582c5)